orderapi/src/test/java/fi/hel/verkkokauppa/order/service/order/OrderServiceTest.java

```diff
@@ -645,4 +644,26 @@ void createOrderWithMerchantId() throws JsonProcessingException {
         orderRepository.save(order);
     }
 
+    @Test
+    @RunIfProfile(profile = "local")
+    public void CreateOrderWithLastValidPurchaseDateTime() {
```

Comment:
You need to add test prefix to test functions ->
`testCreateOrderWithLastValidPurchaseDateTime`

Reply:
I can add the prefix.

Other methods seem to lack the prefix too in the class. While at it, shall I add the prefix to them too?

Reply:
As discussed in slack, I'll skip using the test -prefix and just change the case convention to camel.